### DISCUSSION TITLE
Client Zertifikat für die API

### DISCUSSION THREAD

**Discussion by pbroska** [Digitaler-Impfnachweis/certification-apis#111] [Q&A] [answered]
Welches Zertifikat muss für die API verwendet werden? Das von der SMC-B oder das verlinkte Demozertifikat: https://github.com/Digitaler-Impfnachweis/certification-apis/blob/master/examples/demo-dsc.crt ?

**Comment by ManuelB** [comment, marked as answer]
@pbroska wenn du die PVS API verwendet möchtest, dann musst du das Certificat von der SMC-B an den IdP schicken:

https://youtu.be/oLrjF-OISO0?t=778
